recipes/GKE Setup/create-svc-account.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+SERVICE_EMAIL="$SERVICE_NAME@$GOOGLE_PROJECT.iam.gserviceaccount.com"
```

Comment:
The README should probably mention that `SERVICE_NAME` must be set before invoking. Or perhaps this should accept it as an argument?

Also, this script should definitely validate that it's not empty before proceeding.

Reply:
`GOOGLE_PROJECT`, `BIGQUERY_DATASET`, and `GCS_BUCKET`, too